requirements.txt

```diff
@@ -3,7 +3,6 @@ diraccfg
 autopep8==1.3.3
 coverage
 requests>=2.9.1
-six
```

Comment:
is this file needed at all? I think we should align to use environment.yml like we do for DIRAC? Or none of the 2?

Reply:
removed